### PR TITLE
Add standard sized & named quantum logos, update to use them (#7656)

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

In preparation for the upcoming logo update this PR:
- Adds files for the current browser logo & wordmark that are the same size and file name as the files with the new logo
- Replaces updates current release logos and wordmarks to use those files
  - Except favicons, those will be handled separately

## Issue / Bugzilla link

See #7656

## Testing

- Any 404s?
- Did I delete anything still in use?
- Did I miss anything that needs updates?

---

- http://localhost:8000/ar/firefox/accounts/
- http://localhost:8000/de/firefox/campaign/?xv=berlin
- http://localhost:8000/de/firefox/campaign/?xv=privatsphare
- http://localhost:8000/de/firefox/new/?xv=berlin
- http://localhost:8000/en-US/
- http://localhost:8000/en-US/about/legal/firefox/
- http://localhost:8000/en-US/firefox/
- http://localhost:8000/en-US/firefox/40.0/firstrun/
- http://localhost:8000/en-US/firefox/63.0/whatsnew/all/
- http://localhost:8000/en-US/firefox/64.0/whatsnew/all/
- http://localhost:8000/en-US/firefox/65.0/whatsnew/all/
- http://localhost:8000/en-US/firefox/66.0/whatsnew/all/
- http://localhost:8000/en-US/firefox/67.0.5/whatsnew/all/
- http://localhost:8000/en-US/firefox/67.0.5/whatsnew/all/
- http://localhost:8000/en-US/firefox/67.0/whatsnew/
- http://localhost:8000/en-US/firefox/67.0/whatsnew/all/
- http://localhost:8000/en-US/firefox/67.0/whatsnew/all/
- http://localhost:8000/en-US/firefox/68.0/whatsnew/all/
- http://localhost:8000/en-US/firefox/68.0/whatsnew/all/?v=b
- http://localhost:8000/en-US/firefox/68.0/whatsnew/all/?v=c
- http://localhost:8000/en-US/firefox/69.0/whatsnew/all/
- http://localhost:8000/en-US/firefox/accounts/
- http://localhost:8000/en-US/firefox/accounts/
- http://localhost:8000/en-US/firefox/best-browser/
- http://localhost:8000/en-US/firefox/campaign/
- http://localhost:8000/en-US/firefox/campaign/
- http://localhost:8000/en-US/firefox/campaign/?xv=betterbrowser
- http://localhost:8000/en-US/firefox/campaign/?xv=edge
- http://localhost:8000/en-US/firefox/campaign/?xv=safari
- http://localhost:8000/en-US/firefox/campaign/?xv=safari
- http://localhost:8000/en-US/firefox/channel/desktop/
- http://localhost:8000/en-US/firefox/election/
- http://localhost:8000/en-US/firefox/enterprise/
- http://localhost:8000/en-US/firefox/enterprise/sla/
- http://localhost:8000/en-US/firefox/facebookcontainer/
- http://localhost:8000/en-US/firefox/features/
- http://localhost:8000/en-US/firefox/installer-help/
- http://localhost:8000/en-US/firefox/ios/testflight/
- http://localhost:8000/en-US/firefox/mobile/
- http://localhost:8000/en-US/firefox/new/
- http://localhost:8000/en-US/firefox/switch
- http://localhost:8000/en-US/firefox/switch/
- http://localhost:8000/en-US/firefox/switch/
- http://localhost:8000/en-US/firefox/unsupported-systems/
- http://localhost:8000/en-US/firefox/welcome/1
- http://localhost:8000/en-US/foundation/trademarks/list/
- http://localhost:8000/en-US/privacy/
- http://localhost:8000/en-US/privacy/archive/
- http://localhost:8000/en-US/privacy/archive/firefox/2006-10/
- http://localhost:8000/en-US/privacy/archive/firefox/2009-01/
- http://localhost:8000/en-US/privacy/firefox
- http://localhost:8000/en-US/privacy/firefox-fire-tv/
- http://localhost:8000/en-US/privacy/firefox-private-network/
- http://localhost:8000/en-US/privacy/firefox/
- http://localhost:8000/firefox/67.0/whatsnew/all/
- http://localhost:8000/ru/firefox/new/?geo=ru (disable trailhead switch)

### CODE REVIEW

File: bedrock/privacy/templates/privacy/archive/firefox-2006-10.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 {% block article %}
   <article class="section-content" itemscope itemtype="http://schema.org/Article">
     <header>
-      <img class="logo" src="/media/img/privacy/logo-firefox.png" alt="">
+      <img class="logo" src="/media/img/logos/firefox/quantum/logo-xl.png" alt="">
```
Comment (amychurchwell) [inline bedrock/privacy/templates/privacy/archive/firefox-2006-10.html]: This logo appears blurry to me here, maybe the hi res image helper would help but I wouldn't call it a blocker.

**Comment by stephaniehobson** [comment]
>     http://localhost:8000/media/protocol/img/logos/firefox/og.png
>     http://localhost:8000/protocol/img/logos/firefox/og.png
>     http://localhost:8000/en-US/protocol/img/logos/firefox/og.png

The only one of these that should work is http://localhost:8000/media/protocol/img/logos/firefox/og.png and it's not working for me either :( The others would be bad URLs. 

I opened an issue to add it to protocol-assets: https://github.com/mozilla/protocol-assets/issues/39

File: bedrock/privacy/templates/privacy/archive/firefox-2006-10.html (hunk 1 above)
Comment (stephaniehobson) [reply bedrock/privacy/templates/privacy/archive/firefox-2006-10.html]: Yeah but I just realized these are out dated policies and should have the old-old logo on them. And I do think that is a blocker.

**Comment by stephaniehobson** [comment]
Amy and I connected in person on this and decided:
1) it's okay to merge with mozilla/protocol-assets#39 outstanding because this PR doesn't use that image
2) revert the changes to the privacy pages so that logo update is no longer a blocker to this PR

**Comment by stephaniehobson** [comment]
@amychurchwell Updated to remove the privacy archive pages.

**Pull request merged**